app/controllers/application_controller.rb

```diff
@@ -69,6 +69,10 @@ def unprocessable_entity
     end
   end
 
+  def single_user_mode
+    Rails.configuration.x.single_user_mode && Account.first
```

Comment:
And this could be `@single_user_mode ||= Rails.configuration.x.single_user_mode && Account.first` for multiple calls